addons/website/static/src/snippets/s_image_gallery/000.xml

```diff
@@ -53,7 +53,10 @@
         ========================================================================
     -->
     <t t-name="website.gallery.slideshow.lightbox">
-        <div role="dialog" class="modal o_technical_modal fade s_gallery_lightbox p-0" aria-labbelledby="Image Gallery Dialog">
+        <!-- aria-labbelledby has a typo and is the wrong attribute. It is kept
+             in stable, but is already removed from 16.0 up. -->
+        <div role="dialog" class="modal o_technical_modal fade s_gallery_lightbox p-0"
+                aria-labbelledby="Image Gallery Dialog" aria-label="Image Gallery Dialog" tabindex="-1">
```

Comment:
I asked to be sure if we can modify the XML template and we can (as long as we don't remove/modify anything to not break customizations) so normally we can do it like this :slightly_smiling_face: 
However, they apparently may be cache issues in 15.0 and before, which means that we need to ensure the JS code will still work if it is the old cached template that is used 
So here, it will not crash (since it is just listeners that will never be called) but given the purpose of the PR, I think you should add the `tabindex` attribute in JS instead, to be sure everyone has the fix :slightly_smiling_face: